docs/platforms/javascript/guides/deno/configuration/integrations/breadcrumbs.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Breadcrumbs
+excerpt: ""
+description: "Wraps native APIs to capture breadcrumbs. (default)"
+sidebar_order: 1
+notSupported:
```

Comment:
If you create a page in a guide, you do not longer need to specify for with other platforms/guides is isn't supported. You could remove the `notSupported` list